### PR TITLE
Minor edits to the code.

### PR DESCRIPTION
**Pull request by TheKingy34**
I have put a version number and warning on the bottom screen and other stuff.

### CODE REVIEW

**Comment by TheKingy34** [comment]
Cool. I removed it.

File: source/main.c [hunk 1]
```diff
@@ -96,12 +95,9 @@ int main() {
 				svcSleepThread(5000000000);
 				printf("0004009B00010402.cia\n");
 				svcSleepThread(10000000000);
+			
 			printf("\x1b[31mERROR: \x1b[0mDowngrading failed. Rebooting...\n");
 			svcSleepThread(800000000);
-			//Reboot code
-		        	aptOpenSession();
-		        	APT_HardwareResetAsync(NULL);
-		        	aptCloseSession();
```
Comment (Hamcha) [inline source/main.c]: This should actually work, why was it removed?

**Comment by TheKingy34** [comment]
I had problems compiling it, so I couldn't test it. It would of been cool though.

File: source/main.c (hunk 1 above)
Comment (TheKingy34) [inline source/main.c]: I had problems compiling it, so I couldn't test it. It would of been cool though.

Comment (Hamcha) [inline source/main.c]: Have you tried taking the "NULL" out? I use the same exact code for rebooting on my homebrew and it works without problem.

Comment (Sans0Serif) [inline source/main.c]: Sorry about that! IIRC you need to remove NULL, but I accidentally removed the reboot code and forgot to put it back when I was testing displaying images.

**Pull request merged**